truenas/webui
# NAS-110413 / 21.06 / Fix form issues in data-protection dashboard

Scrub/Snapshot task forms now update form values from correct row.

To test the snapshot or scrub task forms:
- Make sure there are multiple rows in the table
- Ensure that the edit form for each row actually populates all the fields with the correct row data.
- Also make sure that custom select option in form-scheduler works as expected.
